### PR TITLE
fix: inconsistent pagination docstrings

### LINKED ISSUE #8023: Inconsistent docstrings on pagination connection
### Issue

The pagination connection has an inconsistent use of docstrings. It is not clear in the examples below if list and paged are the same thing. 

The pagination connection should have consistent use of terms when being described.

“// List of withdrawals data with corresponding page information”
“// Paged deposits data received with the corresponding page information”
“// Oracle data with page info”

We need to review the docstrings and make these consistent so to avoid confusion when reading about the same thing

### PR DESCRIPTION
**Pull request by guoguojin**
Closes #8023

### CODE REVIEW

**Review by jeremyletang — DISMISSED** [review]

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 1]
```diff
@@ -978,19 +978,19 @@ message ListAccountsRequest {
...
 message AccountsConnection {
-    // List of accounts and their cursors.
+    // Page of accounts data and their corresponding cursors.
     repeated AccountEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// Represents the account with corresponding cursor for it
+// Account data item with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:993]:
```suggestion
// Account data item with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 2]
```diff
@@ -1194,21 +1194,21 @@ message ListAllPositionsRequest {
 
 // Response to query for listing of positions, given the filter is supplied
 message ListAllPositionsResponse {
-    // List of 0 or more positions.
+    // Page of positions data and corresponding page information.
     PositionConnection positions = 1;
 }
 
-// Represents the position data with corresponding cursor for it
+// Position data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:1201]:
```suggestion
// Position data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 3]
```diff
@@ -1387,21 +1387,21 @@ message GetBalanceHistoryRequest {
 
 // Response that is received when querying for balance history
 message GetBalanceHistoryResponse {
-    // List of aggregated balances with the corresponding page information.
+    // Page of aggregated balances with the corresponding page information.
     AggregatedBalanceConnection balances = 1;
 }
 
-// Represents the AggregatedBalanceEdge data with corresponding cursor for it
+// Aggregated balance data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:1394]:
```suggestion
// Aggregated balance data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 4]
```diff
@@ -1585,20 +1585,21 @@ message ListTransfersRequest {
 
 // Response that is received when listing  transfers
 message ListTransfersResponse {
+    // Page of transfers data and corresponding page information.
     TransferConnection transfers = 1;
 }
 
-// Represents the transfers data with a corresponding cursor for it
+// Transfers data with a corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:1592]:
```suggestion
// Transfers data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 5]
```diff
@@ -1683,21 +1684,21 @@ message ListCandleDataRequest {
 
 // Response for list of candles for a market at an interval
 message ListCandleDataResponse {
-    // List of 0 or more candles.
+    // Page of candle data and corresponding page information.
     CandleDataConnection candles = 1;
 }
 
-// Represents the candle data with the corresponding cursor for it
+// Candle data with the corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:1691]:
```suggestion
// Candle data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 6]
```diff
@@ -1766,29 +1767,29 @@ message ListERC20MultiSigSignerAddedBundlesRequest {
 
 // Response from adding a signature bundle to the signer list of a multisig contract for a particular validator
 message ListERC20MultiSigSignerAddedBundlesResponse {
-    // List of bundles for that validator, it may have been added multiple times if removed in between.
+    // Page of bundles for that validator - it may have been added multiple times if removed in between - and corresponding page information.
     ERC20MultiSigSignerAddedConnection bundles = 1;
 }
 
-// Represents the signature data that is to be added with the corresponding cursor for it
+// Signature data that is to be added with the corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:1774]:
```suggestion
// Signature data that is to be added with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 7]
```diff
@@ -1766,29 +1767,29 @@ message ListERC20MultiSigSignerAddedBundlesRequest {
...
 message ERC20MultiSigSignerAddedEdge {
     // Signature data to be added.
     vega.events.v1.ERC20MultiSigSignerAdded node = 1;
     // Cursor that can be used to fetch further pages.
     string cursor = 2;
 }
 
-// Represents the signature bundle data that is to be added the corresponding cursor for it
+// Signature bundle data that is to be added the corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:1782]:
```suggestion
// Signature bundle data that is to be added and the corresponding cursor.
```
Please check this suggestion, the original sentence doesn't read well but my suggestion may not be accurate?

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 8]
```diff
@@ -1825,29 +1826,29 @@ message ListERC20MultiSigSignerRemovedBundlesRequest {
...
+// Signature data to be removed with the corresponding cursor for it
 message ERC20MultiSigSignerRemovedEdge {
     // Signature data to be removed.
     vega.events.v1.ERC20MultiSigSignerRemoved node = 1;
     // Cursor that can be used to fetch further pages.
     string cursor = 2;
 }
 
-// Represents the signature bundle data to be removed, with the corresponding cursor for it
+// Signature bundle data to be removed, with the corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:1841]:
```suggestion
// Signature bundle data to be removed, with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 9]
```diff
@@ -1966,19 +1967,19 @@ message ListTradesRequest {
...
 message TradeConnection {
-    // List of trades data.
+    // Page of trades and their corresponding cursors.
     repeated TradeEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// List of trades
+// Trade data item with corresponding cursor for it.
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:1982]:
```suggestion
// Trade data item with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 10]
```diff
@@ -2041,37 +2042,37 @@ message ListOracleDataRequest {
 
 // Response to get all seen oracle data
 message ListOracleDataResponse {
-    // List of seen oracle data.
+    // Page of seen oracle data and corresponding page information.
     OracleDataConnection oracle_data = 1;
 }
 
-// Oracle specs with cursor
+// Oracle specs data item with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2049]:
```suggestion
// Oracle specs data item with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 11]
```diff
@@ -2041,37 +2042,37 @@ message ListOracleDataRequest {
...
 message OracleSpecsConnection {
-    // Oracle specs.
+    // Page of oracle specs data and their corresponding cursors.
     repeated OracleSpecEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// Oracle data with corresponding cursor
+// Oracle data item with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2065]:
```suggestion
// Oracle data item with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 12]
```diff
@@ -2101,21 +2102,21 @@ message ListMarketsRequest {
 
 // Response from listing markets
 message ListMarketsResponse {
-    // List of markets data.
+    // Page of markets and corresponding page information.
     MarketConnection markets = 1;
 }
 
-// Market data with corresponding cursor
+// Market information with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2109]:
```suggestion
// Market information with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 13]
```diff
@@ -2145,27 +2146,27 @@ message ListPartiesRequest {
 
 // Response from listing parties
 message ListPartiesResponse {
-    // Parties data.
+    // Page of parties data and corresponding page information.
     PartyConnection parties = 1;
 }
 
-// Party data with corresponding cursors
+// Party data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2153]:
```suggestion
// Party data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 14]
```diff
@@ -2145,27 +2146,27 @@ message ListPartiesRequest {
...
 message PartyConnection {
-    // List of parties data.
+    // Page of parties and their corresponding cursors.
     repeated PartyEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// Order data with corresponding cursor
+// Order data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2169]:
```suggestion
// Order data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 15]
```diff
@@ -2204,25 +2205,25 @@ message ObserveMarginLevelsResponse {
...
 message OrderConnection {
-    // Orders data list.
+    // Page of orders and their corresponding cursors.
     repeated OrderEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// Margin data with corresponding cursors
+// Margin data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2216]:
```suggestion
// Margin data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 16]
```diff
@@ -2302,18 +2303,19 @@ message ListEpochRewardSummariesRequest {
...
 message EpochRewardSummaryConnection {
-    // Rewards summary data parts for epochs.
+    // Page of rewards summary data for epochs and their corresponding cursors.
     repeated EpochRewardSummaryEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// Rewards summary data for epoch with corresponding cursors
+// Rewards summary data for epoch with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2318]:
```suggestion
// Rewards summary data for epoch with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 17]
```diff
@@ -2362,21 +2364,21 @@ message ListDepositsRequest {
 
 // Response from listing deposits
 message ListDepositsResponse {
-    // Deposits data.
+    // Page of deposits data and corresponding page information.
     DepositsConnection deposits = 1;
 }
 
-// Deposits data with corresponding cursor
+// Deposits data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2371]:
```suggestion
// Deposits data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 18]
```diff
@@ -2409,21 +2411,21 @@ message ListWithdrawalsRequest {
 
 // Response from listing withdrawals data
 message ListWithdrawalsResponse {
-    // List of withdrawals data parts.
+    // Page of withdrawals data and corresponding page information.
     WithdrawalsConnection withdrawals = 1;
 }
 
-// Withdrawals data with corresponding cursor
+// Withdrawals data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2418]:
```suggestion
// Withdrawals data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 19]
```diff
@@ -2491,20 +2493,21 @@ message ListLiquidityProvisionsRequest {
 
 // Response from listing liquidity provisions
 message ListLiquidityProvisionsResponse {
-    // List liquidity provisions data with corresponding page information.
+    // Page of liquidity provisions data and corresponding page information.
     LiquidityProvisionsConnection liquidity_provisions = 1;
 }
 
-// Paged liquidity provision data with corresponding cursor
+// Liquidity provision data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2500]:
```suggestion
// Liquidity provision data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 20]
```diff
@@ -2573,21 +2576,21 @@ message ListGovernanceDataRequest {
 
 // Response from listing governance data
 message ListGovernanceDataResponse {
-    // Paged governance data with corresponding page information.
+    // Page of governance data and corresponding page information.
     GovernanceDataConnection connection = 1;
 }
 
-// Governance data with corresponding cursor
+// Governance data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2583]:
```suggestion
// Governance data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 21]
```diff
@@ -2619,21 +2622,21 @@ message ListDelegationsRequest {
 
 // Response from listing delegations
 message ListDelegationsResponse {
-    // Delegations data with corresponding page information.
+    // Page of delegations data and corresponding page information.
     DelegationsConnection delegations = 1;
 }
 
-// Delegation data with corresponding cursors
+// Delegation data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2629]:
```suggestion
// Delegation data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 22]
```diff
@@ -2738,21 +2741,21 @@ message ListNodeSignaturesRequest {
 
 // Response to specify the ID of the resource to retrieve aggregated signatures for
 message ListNodeSignaturesResponse {
-    // List of nodes signatures with corresponding page information.
+    // Page of nodes signatures and corresponding page information.
     NodeSignaturesConnection signatures = 1;
 }
 
-// Nodes signature edge with corresponding cursor
+// Nodes signature edge with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2748]:
```suggestion
// Nodes signature edge with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 23]
```diff
@@ -2835,17 +2838,17 @@ message GetNetworkParameterResponse {
   vega.NetworkParameter network_parameter = 1;
 }
 
-// Network parameter with corresponding cursor
+// Network parameter with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2841]:
```suggestion
// Network parameter with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 24]
```diff
@@ -2871,20 +2874,21 @@ message ListCheckpointsRequest {
 
 // Response message containing all checkpoints requested
 message ListCheckpointsResponse {
-    // List of checkpoints data with corresponding page information.
+    // Page of checkpoints data and corresponding page information.
     CheckpointsConnection checkpoints = 1;
 }
 
-// Checkpoint data with corresponding cursor
+// Checkpoint data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2881]:
```suggestion
// Checkpoint data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 25]
```diff
@@ -2906,17 +2910,17 @@ message GetStakeResponse {
     StakesConnection stake_linkings = 2;
 }
 
-// Stake linking data with corresponding cursor
+// Stake linking data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2913]:
```suggestion
// Stake linking data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 26]
```diff
@@ -2983,21 +2987,21 @@ message ListKeyRotationsRequest {
 
 // Response message containing Vega key rotations
 message ListKeyRotationsResponse {
-    // Key rotations data with corresponding page information.
+    // Page of key rotations data and corresponding page information.
     KeyRotationConnection rotations = 1;
 }
 
-// Key rotation data with corresponding cursor
+// Key rotation data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:2994]:
```suggestion
// Key rotation data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 27]
```diff
@@ -3015,19 +3019,19 @@ message ListEthereumKeyRotationsRequest {
...
 message EthereumKeyRotationsConnection {
-    // List of Ethereum key rotations data with corresponding cursors.
+    // Page of Ethereum key rotations data and their corresponding cursors.
     repeated EthereumKeyRotationEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// Ethereum key rotation data with corresponding cursor
+// Ethereum key rotation data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:3034]:
```suggestion
// Ethereum key rotation data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 28]
```diff
@@ -3075,19 +3079,19 @@ message ListProtocolUpgradeProposalsRequest {
...
 message ProtocolUpgradeProposalConnection {
-    // Protocol upgrade proposals page data.
+    // Page of protocol upgrade proposals data and their corresponding cursors.
     repeated ProtocolUpgradeProposalEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// Protocol upgrade proposal data with a corresponding cursor
+// Protocol upgrade proposal data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:3094]:
```suggestion
// Protocol upgrade proposal data with the corresponding cursor.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 29]
```diff
@@ -3103,18 +3107,19 @@ message ListCoreSnapshotsRequest {
...
+// Page of core snapshot data and corresponding page information.
 message CoreSnapshotConnection {
+    // Page of core snapshot data and their corresponding cursors.
     repeated CoreSnapshotEdge edges = 1;
     // Page information that is used for fetching further pages.
     PageInfo page_info = 2;
 }
 
-// Core snapshot data with a corresponding cursor
+// Core snapshot data with corresponding cursor for it
```
Comment (gordsport) [inline protos/sources/data-node/api/v2/trading_data.proto:3122]:
```suggestion
// Core snapshot data with the corresponding cursor.
```

**Review by gordsport — COMMENTED** [review]
Made some suggestions to make the text more consistent with the way we talk about plural cursors

**Review by gordsport — APPROVED** [review]
LGTM

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**